scripts/codechecks/hqDefine.sh

```diff
@@ -0,0 +1,22 @@
+# run as
+# bash scripts/codechecks/hqDefine.sh
```

Comment:
just curious - why no `#!/bin/bash` and `chmod +x scripts/codechecks/hqDefine.sh`?